pages/spicedb/concepts/commands.mdx

```diff
@@ -28,8 +28,11 @@ The following are configuration flags that can be provided to every SpiceDB comm
 | Flag | Description | Default |
 | --- | --- | --- |
 | --datastore-conn-uri | connection string used by remote datastores (e.g. "postgres://postgres:password@localhost:5432/spicedb") |  |
+| --datastore-credentials-provider-name | retrieve datastore credentials dynamically using aws-iam |  |
 | --datastore-engine | type of datastore to initialize ("cockroachdb", "mysql", "postgres", "spanner") | `memory` |
 | --datastore-mysql-table-prefix | prefix to add to the name of all SpiceDB database tables |  |
+| --datastore-read-replica-conn-uri | connection string used by remote datastores for read replicas (e.g. \"postgres://postgres:password@localhost:5432/spicedb\"). Only supported for postgres and mysql. |  |
```

Comment:
this flag is duplicated 2 times in this PR.

What does "remote datastores" mean?

We need to clarify this is a list of connection URIs, i.e. a comma-separated list of URIs. 

This is something we should probably address in the generated flag comment as well, but it's VERY important to note:
- in the case of MySQL, customer must list MySQL replica host URIs and NEVER use a load-balancer URI
- Postgres can be pointed to either a load-balancer or replica hosts URIs

Reply:
updated!